pilot/cmd/istioctl/main.go

```diff
@@ -481,14 +489,23 @@ istioctl context-create --api-server http://127.0.0.1:8080
 	}
 )
 
+const defaultKubeConfigText = "$KUBECONFIG else $HOME/.kube/config"
```

Comment:
This isn't completely accurate on windows. You'd need to include $HOMEDIR, $HOMEPATH, and $USERPROFILE as well.

Reply:
That said, https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#explore-the-homekube-directory assumes `$HOME` and the non-windows case so maybe its fine for doc purposes.

Reply:
yes I think it's incrementally better in this PR for most users and just nice that it also works on windows

Reply:
It'll be real fun for i18n...